startup/30-detectors_and_cameras_common.py

```diff
@@ -183,7 +198,200 @@ def _num_captured_changed(self, value=None, old_value=None, **kwargs):
             self._save_started = False
 
 
-class XPDContinuous(ContinuousAcquisitionTrigger, XPDAreaDetector):
+class XPDAreaDetector_tiff(ContinuousAcquisitionTrigger, XPDAreaDetector_tiff):
```

Comment:
This class uses the same-named class as a base class. I think it's worth separating those to avoid ambiguity.